src/components/FormValidator.js

```diff
@@ -54,6 +54,11 @@ export default class FormValidator {
     }
   }
 
+  disableButton() {
+    this._submitButton.classList.add(this._inactiveButtonClass);
+    this._submitButton.disabled = true;
```

Comment:
Please, use `disableButton` in line 49 not to duplicate the same code of disabling